addons/gatekeeper/gatekeeper.yaml

```diff
@@ -34,15 +33,10 @@ spec:
   chartReference:
     chart: gatekeeper
     repo: https://mesosphere.github.io/charts/staging
-    version: 0.4.2
+    version: 0.5.0
     values: |
       ---
+      replicas: 2
       image:
-        tag: v3.0.4-beta.1
-      resources:
-        limits:
-          cpu: 500m
-          memory: 500Mi
-        requests:
-          cpu: 200m
-          memory: 300Mi
```

Comment:
Unless these are the new chart defaults, we include these as our opinionated choices.

Reply:
There are new chart defaults https://github.com/mesosphere/charts/blob/master/staging/gatekeeper/values.yaml#L56. I am open-minded if we want to keep these vs chart defaults. I don't have enough data point about how we arrived at those values vs upstream. In the absence of that data point, I was favoring upstream.

Reply:
these were probably arrived at via MWT. I would suggest we keep these values for now and dial them up/down based on the next mwt run.

Reply:
The values were extracted by our learnings to keep gatekeeper running healthy while doing load tests.

Reply:
It seems the code of gatekeeper changed now. We are not going to exercise it during this new MWT. So we can keep the defaulted ones in the chart. The gatekeeper initial resource allocation were lower before, so we experienced OOM issues during our MWT tests.

Reply:
Sounds good.